src/app/governance/utils/getCalendarEvents.ts

```diff
@@ -0,0 +1,12 @@
+import { CalendarEventsSchema } from '../schemas/CalendarEventSchemas'
+
+export async function getCalendarEvents(endpoint: string) {
+  const response = await fetch(endpoint)
+
+  if (!response.ok) {
+    throw new Error('getEvents: Failed to fetch events')
```

Comment:
Should we also add `Sentry.captureError`?

Reply:
Will add it to our followup dev work ticket 🙏🏼